docs/docs/APIGuide/MLPipeline/DLEstimator/DLEstimator.md

```diff
@@ -0,0 +1,84 @@
+## DLEstimator ##
+
+**Scala:**
+
+```scala
+val estimator = new DLEstimator(model: Module[T], criterion: Criterion[T], val featureSize: Array[Int], val labelSize: Array[Int])
+```
+
+Source code [here](https://github.com/intel-analytics/BigDL/blob/585818a3fb0e7339eb4e3831f08da82b7d5e47ba/spark/dl/src/main/scala/org/apache/spark/ml/DLEstimator.scala#L53).
+

```

Comment:
Make it clear in the beginning that DLEstimator extends MLlib Estimator and can be directly used in Spark ML pipelines (and add a link to the Spark ML pipelines doc).